jf_agent/main.py

```diff
@@ -263,9 +260,32 @@ def main():
                     return True
 
                 if config.run_mode_includes_download:
+                    ingest_config = get_ingest_config(
+                        config=config,
+                        creds=creds,
+                        endpoint_jira_info=jellyfish_endpoint_info.jira_info,
+                        endpoint_git_instances_info=jellyfish_endpoint_info.git_instance_info,
+                        jf_options=jellyfish_endpoint_info.jf_options,
+                    )
+                    # Mark some agent upload directories as skippable during out
+                    # data uploading phase, because they already get uploaded via
+                    # JF Ingest
+                    from jf_agent.git.utils import JF_INGEST_SUPPORTED_PROVIDERS
+
+                    directories_to_skip_uploading_for.update(
+                        [
+                            f'git_{git_config.instance_file_key}'
+                            for git_config in ingest_config.git_configs
+                            if git_config.git_provider.value.lower()
+                            in JF_INGEST_SUPPORTED_PROVIDERS
+                        ]
+                    )
+                    # Jira is supported by all customers, always skip it
+                    directories_to_skip_uploading_for.add('jira')
```

Comment:
Q: Previously, were we dual-submitting, or is this due to changes in jf_ingest version bump?

Reply:
Yeah, there was a bug where we were double submitting `github` data. Not really a huge deal, but better practice to not do it. It might have had some run time performance if a customer tried to upload a massive file? But this is SUPER threaded so it was likely minimal